chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -47,7 +47,7 @@
 
   bash "disable_openstack_services" do
     code <<-EOF
-      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1) \
+      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1 | grep -Ev 'keystone|neutron') \
```

Comment:
As mentioned elsewhere, in the HA case this should be a no-op because [they're already disabled](https://github.com/crowbar/crowbar-ha/blob/master/chef/cookbooks/crowbar-pacemaker/providers/service.rb#L41). However it's worth doing this anyway just in case.

But why aren't you disabling `keystone` and `neutron`?